### PR TITLE
Removed equal heights from spotlight and magazine card grids

### PR DESCRIPTION
**Pull request by imonroe**
# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- Lots of stuff related to the magazine curtain, miscellaneous styles, and working tests with coverage for custom blocks and the migration process plugin.

# Review By (Date)
- asap

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Urgency
- How urgent is this? (Normal, High)

# Review Tasks

## Setup tasks and/or behavior to test

1. This code is already on Dev, so you can test the functionality there.  
2. Verify tests pass.

### Site Configuration Sync

- very likely should do a `drush cim` to get these changes.

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: modules/custom/engineering_profile_helper/modules/engineering_magazine/templates/magazine-curtain-block.html.twig [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+	<div class="curtain-image">
+		<img src="{{ curtain_content.media_url }}"/>
+	</div>
+
 	<div class="curtain-content">
-		<h1>This is my curtain.</h1>
 
-		<p>Curtain content:
-			{{ curtain_content }}</p>
+		<a href="{# curtain_content.issue_url #}" class="engineering-accent-color__background">
```
Comment (mdyoung3) [inline modules/custom/engineering_profile_helper/modules/engineering_magazine/templates/magazine-curtain-block.html.twig:9]: Is it okay this href has commented out variable in it?

**Review by mdyoung3 — COMMENTED** [review]
Overall looks like good work. I have one question, and I have a push removing a duplicate bit in composer.json.

File: modules/custom/engineering_profile_helper/modules/engineering_magazine/templates/magazine-curtain-block.html.twig (hunk 1 above)
Comment (imonroe) [reply modules/custom/engineering_profile_helper/modules/engineering_magazine/templates/magazine-curtain-block.html.twig:9]: good catch.

Comment (imonroe) [reply modules/custom/engineering_profile_helper/modules/engineering_magazine/templates/magazine-curtain-block.html.twig:9]: fixed.

**Review by mdyoung3 — APPROVED** [review]
Test passed.

**Pull request merged**